app/models/bill.rb

```diff
@@ -13,7 +13,7 @@ class Bill < ApplicationRecord
 
   scope :find_all_by_account_id, ->(account_id){where account_id: account_id}
   scope :order_by_created_at, ->{order created_at: :desc}
-  scope :by_status, ->(status){where status: status}
+  scope :by_status, ->(status){status == "all" ? all : where(status: status)}
```

Comment:
```suggestion
  scope :by_status, ->(status){where(status: status) if status.present?}
```
ở chỗ selectbox em truyền giá trị xún server là "" là dc á cu

Reply:
dạ dạ